modules/openthread/platform/radio.c

```diff
@@ -524,6 +524,13 @@ static void openthread_handle_frame_to_send(otInstance *instance,
 		goto exit;
 	}
 
+	if (IS_ENABLED(CONFIG_OPENTHREAD)) {
```

Comment:
isn't `platform` subdir added in cmake only when `CONFIG_OPENTHREAD` is enabled? 
Also, I think using `#if defined(CONFIG_<kconfig>)` guard could be a better option here?

Reply:
True, this check seems redundant.

Reply:
Unfortunately, there are tests where `radio.c` is built without OT stack and the the build fails because undefined reference so it seems we need this check here.

@maciejbaczmanski regarding `IS_ENABLED(...)` - from the experience with previous contributions this is a preferred way of config checking when there are no obstacles like conditionally compiled structure members etc.